### PR TITLE
Adding kube-proxy and coreDNS files for EKS ARM instances

### PR DESCRIPTION
**Pull request by nithu0115**
*Issue #, if available:* N/A

*Description of changes:*

Adding kube-proxy and coreDNS manifest files for EKS ARM instances

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: preview-programs/eks-ec2-a1-preview/kube-proxy-arm-1.14.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                operator: In
+                values:
+                - arm64
+      hostNetwork: true
+      tolerations:
+      - operator: "Exists"
+      priorityClassName: system-node-critical
+      containers:
+      - name: kube-proxy
+        image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/kube-proxy-arm64:v1.13.10
```
Comment (vipulsabhaya) [inline preview-programs/eks-ec2-a1-preview/kube-proxy-arm-1.14.yaml]: should be v1.14.8

File: preview-programs/eks-ec2-a1-preview/dns-arm-1.14.yaml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+              topologyKey: kubernetes.io/hostname
+            weight: 100
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      - key: "CriticalAddonsOnly"
+        operator: "Exists"
+      containers:
+      - name: coredns
+        image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/coredns-arm64:v1.2.6
```
Comment (vipulsabhaya) [inline preview-programs/eks-ec2-a1-preview/dns-arm-1.14.yaml]: we need to update this to match EKS 1.14.

**Comment by tabern** [comment]
@nithu0115 can you take a look at this and push an update per @vipulsabhaya s comments?

**Comment by nithu0115** [comment]
@taber, sure. I will update the images and submit near PR as this one is already merged!

**Pull request merged**